sdk/storage/azfile/directory/client_test.go

```diff
@@ -219,63 +216,6 @@ func (d *DirectoryRecordedTestsSuite) TestDirectoryCreateNegativeMultiLevel() {
 	testcommon.ValidateFileErrorCode(_require, err, fileerror.ParentNotFound)
 }
 
-func (d *DirectoryUnrecordedTestsSuite) TestDirectoryClientUsingSAS() {
```

Comment:
why are we deleting these, we can just have it as live only?

Reply:
Service SAS at directory doesn't work. It works for share and file only.
https://learn.microsoft.com/en-us/rest/api/storageservices/create-service-sas#permissions-for-a-file

So, I have removed the SAS creation method in the directory client and also removed the tests related to it. If you check this test case, you can see that the service returns authentication error when directory SAS is used.